Include/cpython/code.h

```diff
@@ -226,11 +226,15 @@ static inline Py_ssize_t PyCode_GetNumFree(PyCodeObject *op) {
     return op->co_nfreevars;
 }
 
-static inline int PyCode_GetFirstFree(PyCodeObject *op) {
+static inline int PyUnstable_Code_GetFirstFree(PyCodeObject *op) {
     assert(PyCode_Check(op));
     return op->co_nlocalsplus - op->co_nfreevars;
 }
 
```

Comment:
If the deprecation has no detail, for me, it can be removed in Python 3.15. But the doc is more specific:

"The old name is deprecated, but will remain available until the signature changes again."

Please add a comment to explain it here.

Reply:
> If the deprecation has no detail, for me, it can be removed in Python 3.15.

What are you basing this on?

Reply:
PEP 387: Similarly a feature cannot be removed without notice between any two consecutive releases.

Reply:
I'm sorry, I didn't disable automerge after your comment. I didn't mean to ignore you :(


But IMO, the “Basic Policy” is just a summary. The [“Making Incompatible Changes”](https://peps.python.org/pep-0387/#making-incompatible-changes) section has more detailed instructions.
If you disagree with my reading, let's take it to Discourse; this isn't a good place for it.

Reply:
It's just that this function has a specific comment in the doc, IMO it's worth it to repeat it in the header to avoid future eagger removal. But well, now that the change is merged, it's maybe no longer worth it.